source/MapDetailPanel.cpp

```diff
@@ -587,17 +589,17 @@ void MapDetailPanel::DrawKey()
 	{
 		// Each system is colored by the government of the system. Only the
 		// four largest visible governments are labeled in the legend.
-		vector<pair<double, const Government *>> distances;
-		for(const auto &it : closeGovernments)
+		vector<pair<unsigned int, const Government *>> screenNum;
+		for(const auto &it : GovernmentCounts)
```

Comment:
variable names should start with a lowercase letter
```suggestion
		for(const auto &it : governmentCounts)
```